recipe-box-frontend/src/main/frontend/src/components/RecipesList/Categories.css

```diff
@@ -0,0 +1,18 @@
+button,
+input {
+    border: none;
+    background-color: transparent;
+    width: 100%;
+    font-size: large;
+    font-weight: bold;
+    margin-bottom: 10px;
+    box-sizing: border-box;
+    text-align: left;
+    margin-top: 50px;
+}
+
+.categories {
+    display: flex;
+    margin-left: 215px;
+    margin-right: 220px;
+}
```

Comment:
For margins and paddings I have a few suggestions. 
First of all, in this example you have margin right and left for the categories and also for the recipe grid. You should add those to the parent class, instead of adding them separately. 
Second, if you use more than one margin value I would suggest using the shorthand property for exapmple:
 margin: 25px 50px 75px 100px;
            top margin is 25px
            right margin is 50px
            bottom margin is 75px
            left margin is 100px